xml/System.Data/DataReaderExtensions.xml

```diff
@@ -389,13 +427,30 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="reader">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="T">The type of the value to be returned.</typeparam>
+        <param name="reader">The <see cref="T:System.Data.Common.DbDataReader" /> to get the column value from.</param>
+        <param name="name">The name of the column.</param>
+        <param name="cancellationToken">An optional <see cref="T:System.Threading.CancellationToken" /> to cancel the asynchronous operation.</param>
+        <summary>Asynchronously gets the value of the specified column as the requestsed type.</summary>
+        <returns>The value of the specified column.</returns>
```

Comment:
This async, Returns a Task<T\>. Better way to refer to it?

Reply:
We should probably see how other BCL async APIs document this and align with that...

Reply:
Yeah, but I was getting too tired to look :smile: Perhaps someone knows this from memory.

Reply:
The best way to refer to it is, "A task whose <see cref="P:System.Threading.Tasks.Task`1.Result" /> property contains the value of the specified column.